_includes/2017/codefund.html

```diff
@@ -1,4 +1,7 @@
 <aside class='codefund-sponsor' data-js-no-preview>
-  <script src='https://codefund.io/scripts/{{ site.data.codefund.token }}/embed.js?template=centered'></script>
-  <div id='codefund_ad'></div>
+  <div id="codefund"><!-- fallback content --></div>
+  <script src="https://codefund.io/properties/51/funder.js" async="async"></script>
```

Comment:
If you change this to `https://codefund.app/properties/51/funder.js`, it will not be blocked by ad blockers.